client/src/modules/stock/inventory-adjustment/inventory-adjustment.js

```diff
@@ -27,14 +27,23 @@ function StockInventoryAdjustmentController(
   vm.movement = {};
   vm.stockOut = {};
 
+  vm.currentInventories = [];
+  vm.overconsumption = [];
+
   vm.onDateChange = date => {
     vm.movement.date = date;
     loadInventories(vm.depot);
+    loadCurrentInventories(vm.depot);
+
+    vm.overconsumption = [];
   };
 
   vm.onChangeDepot = depot => {
     vm.depot = depot;
     loadInventories(vm.depot);
+    loadCurrentInventories(vm.depot);
+
```

Comment:
Same question as before: How does this handle cases where the already existing data has negatives?

Reply:
Products with negative quantities will be considered as out of stock products.
![image](https://user-images.githubusercontent.com/8035588/112832518-570a1880-908d-11eb-858e-617d68c1cae9.png)

Reply:
Do we have a way to fix the lots with negative quantities?
Since lots with negative quantities will not be displayed, how do we find them and fix them?

Reply:
![image](https://user-images.githubusercontent.com/8035588/112985356-fa276480-9157-11eb-8aee-a3b9baa8b1a0.png)
The stock adjustment module allows you to do this, you just have to check the option **Include exhausted or negative lots**

Reply:
Thanks for pointing this out.  I had forgotten about it.